irichardo/Learnify
# component form first changes, more changes last; & validations update…

### Component From
_branch form extended_

- form component
- validation updates
- see more changes last

1. style for error message
2. change styles and content this page
